### ISSUE TITLE
pushing illegal key makes menu invisible

### ISSUE THREAD

**Issue by memmerto** [MarcNo/lifelines#48]
Versions affected: 3.0.8 &amp;amp; 3.0.7 at least

Pushing an illegal key \(eg, pushing &amp;quot;l&amp;quot; in the
utilities menu\) makes the menu vanish, but the
menu is still active, which is confusing. The
workaround is to push &amp;quot;q&amp;quot; to exit the menu and
get painting back on track.

The problem is that place\_std\_msg does a wrefresh
on the main\_win to put that message out on the
bottom line, which causes the main window to get
repainted, losing the windows in front of it.

The fix is to call touch\_all instead, if appropriate.

Reported by: puddle

Original Ticket: "bugs/48":https://sourceforge.net/p/bugs/48
